scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -220,6 +228,10 @@ def image(self) -> str:
     @property
     def code_dir(self) -> str:
         """Component code directory."""
+        if(self._yaml.get('type') == ComponentType.PARALLEL.value):
+
+            task = self._yaml.get('task')

```

Comment:
Sorry, GitHub added that extra blank line. Also, reformatted `if` for consistency with the rest of the file.
```suggestion
        if self._yaml.get('type') == ComponentType.PARALLEL.value:
            task = self._yaml.get('task')
```